package.json

```diff
@@ -1,12 +1,17 @@
 {
   "name": "front-assesment",
-  "version": "0.0.0",
+  "version": "0.0.1",
   "scripts": {
     "ng": "ng",
     "start": "ng serve",
+    "start:api": "cd api && npm start",
     "build": "ng build",
     "watch": "ng build --watch --configuration development",
-    "test": "ng test"
+    "test": "jest",
+    "test:coverage": "jest --coverage",
+    "test:e2e": "cd e2e && npm run cypress",
+    "test:e2e:local:ci": "cd e2e && npm run cy:local:ci",
+    "test:e2e:ci": "cd e2e && npm run cy:ci"
```

Comment:
In a pipeline, this is not going to work, because the app is not started. You need to add ``start-server-and-test`` dependency and configure it here. That dependency starts the app, and stops it after executing the tests.